eosdac-material/README.md

```diff
@@ -1,4 +1,10 @@
-eosDAC Toolkit GUI
+#Purpose
+A frontend user interface to the eosDAC Member Client web application.
+
+# Technical Details
+The member's client uses **Vue.js** as the frontend framework and **Quasar** as the server side rendering framework which then connects to the EOS mainnet (depending on the configuration file). In order to change the Member Client to interface with you DAC, you want to follow the instructions and general guidelines below.
```

Comment:
```suggestion
The member's client uses **Vue.js** as the frontend framework and **Quasar** as the server side rendering framework which then connects to the EOS mainnet (depending on the configuration file). In order to change the Member Client to interface with your DAC, you want to follow the instructions and general guidelines below.
```

Reply:
we don't do server side rendering. quasar is a framework on top of vue.

Reply:
Ah okay. It wasn't clear when I looked that up on the interwebs. Thanks @kasperfish